core/services/flux_monitor_helpers_test.go

```diff
@@ -29,7 +29,7 @@ func (p *PollingDeviationChecker) ExportedRespondToNewRound(log eth.Log) error {
 }
 
 func (p *PollingDeviationChecker) ExportedPoll() error {
-	return p.poll()
+	return p.poll(p.threshold)
```

Comment:
Seems weird to call `p.poll()` with the argument `p.threshold` since it's accessible inside the callee function?

Reply:
@samphilipd I think this is parameterized because it's called with 0 in an error condition https://github.com/smartcontractkit/chainlink/pull/2233/files#diff-e2a5bfd429ed26127568dfa7da0a4042R351